apps/src/sites/studio/pages/levels/show.js

```diff
@@ -61,14 +61,21 @@ function initPage() {
 
   if (hasScriptData('script[data-aitutordata]')) {
     const aiTutorData = getScriptData('aitutordata');
-    const {levelId, type, hasValidation, isAssessment, isProjectBacked} =
-      aiTutorData;
+    const {
+      levelId,
+      type,
+      hasValidation,
+      isProjectBacked,
+      aiTutorAvailable,
```

Comment:
`aiTutorAvailable` is the flag coming from the backend? Sorry, I'm getting used between `isAvailable` and `aiTutorAvailable` 😬

Reply:
Yes, `aiTutorAvailable` is the name of the variable that gets defined in the `show.html.haml` using the helper method I added to the `level.rb` model